.golangci.yml

```diff
@@ -0,0 +1,120 @@
+# Tested with golangci-lint ver. 1.37
+run:
```

Comment:
`run` key is already present on [line 114](https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/293/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R114). This duplication inhibits the `timeout :3m` spec and force you to add `args: --timeout 3m0s`  to the action job definition.

Reply:
Fixed, thanks